test/e2e/conformance_test.go

```diff
@@ -57,6 +57,7 @@ var _ = Describe("conformance tests", func() {
 	Measure(specName, func(b Benchmarker) {
 
 		name := fmt.Sprintf("cluster-%s", util.RandomString(6))
+		setEnvVar("USE_CI_ARTIFACTS", "true", false)
```

Comment:
Why the override on the env variable, wouldn't we want to use the value set from Prow?

Reply:
It's coming in as an explicit command line argument `--use-ci-artifacts`, and setting env var for the purpose of envsubst rendering from the CAPI package.

Maybe should have wired up differently